content/academy/pro_apify_scraper_development/solutions/actor_building.md

```diff
@@ -18,7 +18,7 @@ apify create demo-actor
 
 Once you run this command, you'll get prompted into a menu which you can navigate using your arrow keys. Each of these options will generate different boilerplate code when selected. We're going to work with CheerioCrawler today, so we'll select that one, then press <p><kbd>Enter</kbd><p>.
```

Comment:
The `<p>` tag messes up the `<kbd>` tag.

<img width="242" alt="Screenshot 2022-04-20 at 13 18 24" src="https://user-images.githubusercontent.com/38246758/164219375-5d86e12b-482d-4f1d-90ba-8ced21441c19.png">

Reply:
The `<kbd>` tag doesn't work then. It wasn't working standalone, so I will change all of these tags to bold instead.